.devcontainer/post-create.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+DIR=`pwd`
+echo "Current directory: $DIR"
+
+nvm use
+
+# Install the root package dependencies
+npm install
+
+# Install the API package dependencies
+cd $DIR/packages/api
+npm install
+
+# Install the API package dependencies
+cd $DIR/packages/webapp
+pnpm install
+
+# Install the shared package dependencies
+cd $DIR/packages/shared
+npm install
+
+# Copy the default environment files
+cp $DIR/packages/api/.env.default $DIR/packages/api/.env
+cp $DIR/packages/webapp/.env.default $DIR/packages/webapp/.env
+
+if [ -x "$(command -v docker-compose)" ]; then
+  docker-compose up --detach
+else
+  docker compose up --detach
+fi
```

Comment:
Could you please explain what this block should be doing? It seems it does the same in both cases of the condition

Reply:
In my computer (Debian Bookworm) there is command `docker-compose` and when I try to run `docker compose` (without `-`) then I have an error:
```
$ docker compose
docker: 'compose' is not a docker command.
```
On the other hand in GitHub Codespaces there is command `docker compose`.

Command `docker-compose` is the old docker script command and as I know it is not maintained anymore but exists on some systems (like mine Debian).
Command `docker compose` is the new docker script and should be use.

In summary this is only for backward compatibility.

I use this script to start development on my local computer (it works not only in GitHub Codespaces).
Of course it can be removed in favour of the new version `docker compose`.

(_I'm not sure why Debian Bookworm has such old version of Docker - `20.10` vs current `27`._)